src/ImapLibrary2/__init__.py

```diff
@@ -184,6 +184,14 @@ def get_multipart_content_type(self):
         """
         return self._part.get_content_type()
 
+    def get_multipart_param(self, param:str):
+        """Returns the content type parameter ``param`` of current part of selected multipart email message.
```

Comment:
This comment is wrong. The keyword returns any parameter, not just content type. Should add a proper example where this is used to return the content type param though. Now the example is not correct.

Reply:
Would be better to allow returning any header. See the doc there "Optional header if given, specifies the message header to use instead of Content-Type."

But also this anyway needs a proper example to doc if added, and maybe better clear name.